### DISCUSSION TITLE
[Feature Request] JetBrains: Customize color

### DISCUSSION THREAD

**Discussion by leizhag** [community/community#7628] [Copilot Conversations] [answered]
The color used in JetBrains plugin feels like 8C8C8C. Can I adjust it to lighter so that it's more distinguishable with normal code? This'll improve the accessibility.

**Comment by jansorg** [comment, marked as answer]
Thanks! The current color is "comment text color" of the current theme. We're tracking this internally now.

**Reply by leizhag** [reply]
<img width="1094" alt="截屏2021-11-12 下午7 39 48" src="https://user-images.githubusercontent.com/18711879/141461738-a2a39993-ee23-4c5f-85f1-c8d81ad46ecb.png">
Thanks for your reply. I tried changing these three, but the code color by Copilot remains unchanged.

**Reply by jansorg** [reply]
I'm sorry, my first post wasn't entirely correct. It's actually "Inlay hints: text without background".
![Screenshot_20211112_151039](https://user-images.githubusercontent.com/11473063/141480404-8e31e488-b4ae-48fd-a2c3-1a5c56d3dca1.png)

**Reply by jansorg** [reply]
fyi, the latest update (1.1.0) now has a setting to define a custom color for the inlay hint. Your feedback is appreciated! :)

**Comment by soulofmischief** [comment]
@jansorg seems there has been a regression, and now the color and weight is indistinguishable from the normal text style regardless of this setting.

**Reply by jansorg** [reply]
cc @bmuskalla, who's in charge now

**Reply by wbwlife** [reply]
@jansorg sametrouble!

The color for current line code completions is so bright,  it is indistinguishable from the my normal code now!

**Reply by rlnt** [reply]
It is very horrible UX now. It's completely white for me and it's using a color scheme entry that would change a lot of other colors too. Very bad change IMO.
This wouldn't be a problem if the option for the color would work in the Copilot settings.

**Reply by bmuskalla** [reply]
Thanks for reporting this. There indeed seems to be a regression how we handle the styles. We'll release a new version with a fix in the next few hours.

**Reply by sbc640964** [reply]
Indeed, it is really disturbing,
I'm leaving the keyboard for a moment, and I don't understand who wrote to me here...  :) :(

**Reply by soulofmischief** [reply]
@bmuskalla Thanks a ton! Appreciate the rapid turnaround.
